src/curves/ed25519.ts

```diff
@@ -0,0 +1,19 @@
+import { getPublicKey as getEd25519PublicKey } from '@noble/ed25519';
+
+export { CURVE as curve } from '@noble/ed25519';
+
+export const name = 'ed25519';
+export const secret = Buffer.from('ed25519 seed', 'utf8');
+
+// All private keys are valid for ed25519
+export const isValidPrivateKey = () => true;
```

Comment:
For documentation sake - this should take a single unused parameter, the key

Reply:
If you directly call `ed25519.isValidPrivateKey` it would, but when doing something like
```ts
const curve: Curve = ed25519;
curve.isValidPrivateKey(key);
```
the parameter just gets ignored. But I will add it anyway just for documentation purposes.